MethodDecorator.Fody/ModuleWeaver.cs

```diff
@@ -25,6 +25,15 @@ public class ModuleWeaver {
 
         this.DecorateDirectlyAttributed(decorator);
         this.DecorateAttributedByImplication(decorator);
+
+        if(this.ModuleDefinition.AssemblyReferences.Count(r => r.Name == "mscorlib") > 1) {
```

Comment:
Before, it would silently write the assembly and the developer would not even know that there is a second mscorlib added. I added this check here in order to fail the rewrite process instead of producing a corrupted assembly. That will fail every unit test and will be useful to guard against someone changing the IL in a way that causes Cecil to import a second mscorlib. 
I am not sure if this is clear but MethodDecorator before this PR would corrupt .net2 assemblies and portable libraries. The portable library would not fail when you test it on a system which has the full .net 4 but will fail to load on a system that does not have it.